UnityProject/Assets/Scripts/Core/Input System/InteractionV2/TileInteraction/ElectrifiedGrilleInteraction.cs

```diff
@@ -11,7 +13,7 @@ namespace Tiles
 	/// </summary>
 	[CreateAssetMenu(fileName = "ElectrifiedGrilleInteraction",
 		menuName = "Interaction/TileInteraction/ElectrifiedGrilleInteraction")]
-	public class ElectrifiedGrilleInteraction : TileInteraction
+	public class ElectrifiedGrilleInteraction : TileInteraction, IBumpableObject
```

Comment:
Issue if you make a offshoot cable, and don't have any resistance at the end of it it won't be electrified